### DISCUSSION TITLE
its possible to handle urls ?

### DISCUSSION THREAD

**Discussion by dcollaoa** [GooRoo/mkdocs-obsidian-bridge#16] [Q&A]
Hey, nice work!... Im looking for handle with urls because I have a lot of paths on my doc, let me send you my screenshot:

![chrome_RI93O6D3Qy](https://github.com/user-attachments/assets/30d81610-831e-46fc-ac39-15e85bf17f36)

Thanks!

**Comment by GooRoo** [comment]
Hey @dcollaoa,
Could you please elaborate on what exactly you are trying to achieve?
